docs/tutorials/ui-customization.md

```diff
@@ -104,6 +106,19 @@ customized with `addSeekBar` and `addBigPlayButton` booleans in the config.
 UI layout can be reconfigured at any point after it's been created.
 Please note that custom layouts might need CSS adjustments to look good.
 
+#### Configuring Statistics
+
+The list of statistics that are displayed when toggling the statistics button can be customized by specifing a `statisticsList` on the configuration. By default, all of the following are activated.
```

Comment:
It'd be helpful to link to the `Stats` extern, which is what this is displaying.
E.g.: `This can display any value from the player stats. See {@link shaka.extern.Stats here} for a full list of options.`

Reply:
Thank you Theodab, I linked the extern on the refactor!